data_safe_haven/infrastructure/components/wrapped/nfsv3_storage_account.py

```diff
@@ -0,0 +1,68 @@
+from collections.abc import Mapping, Sequence
+
+from pulumi import Input, Output, ResourceOptions
+from pulumi_azure_native import storage
+
+from data_safe_haven.external import AzureIPv4Range
+
+
+class NFSV3StorageAccount(storage.StorageAccount):
```

Comment:
Prefer the name to be of the form `WrappedX` for clarity.

Reply:
What would you call it?

I didn't want to call it `WrappedStorageAccount` because it isn't really that general. It feels to me more like a specific archetype of a storage account that we use.

`WrappedNFSV3StorageAccount`?

Reply:
I think `WrappedNFSV3StorageAccount` is OK for now - we can add a more generic `WrappedStorageAccount` as an intermediate layer if/when we write a similar abstraction for the other storage accounts that we're using.